gnomad/utils/vep.py

```diff
@@ -619,19 +620,26 @@ def filter_vep_transcript_csqs(
     :param vep_root: Name used for VEP annotation. Default is 'vep'.
     :param synonymous: Whether to filter to variants where the most severe consequence is 'synonymous_variant'. Default is True.
     :param canonical: Whether to filter to only canonical transcripts. Default is True.
+    :param mane_select: Whether to filter to only MANE Select transcripts. Default is False.
     :param filter_empty_csq: Whether to filter out rows where 'transcript_consequences' is empty, after filtering 'transcript_consequences' to the specified criteria. Default is True.
     :return: Table or MatrixTable filtered to specified criteria.
     """
-    if not synonymous and not canonical and not filter_empty_csq:
+    if not synonymous and not (canonical or mane_select) and not filter_empty_csq:
         logger.warning("No changes have been made to input Table/MatrixTable!")
         return t
 
+    if canonical and mane_select:
```

Comment:
I suppose, you could have both set to True, it would filter to only transcripts that are both the MANE and canonical